src/components/DiagnosticsBody.vue

```diff
@@ -26,37 +40,40 @@ export default Vue.extend({
           label: 'Category',
         },
       ],
-      rows: [
-        {
-          id:            0,
-          description:   'The ~/.rd/bin directory has not been added to the PATH, so commandline utilities are not configured in your back shell',
-          documentation: 'https://docs.rancherdesktop.io/',
-          category:      'Kubernetes',
-          mute:          false,
-          fixes:         { description: 'You have selected manual PATH configuration, you can let Rancher Desktop automatically configure it.' },
-        },
-        {
-          id:            1,
-          description:   'Are the files under ~/.docker/cli-plugins symlinks to ~/.rd/bin?',
-          documentation: 'https://docs.rancherdesktop.io/',
-          category:      'Kubernetes',
-          mute:          false,
-          fixes:         { description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sit amet iaculis diam. Nullam ut dolor nec dolor vestibulum viverra id a arcu.' },
-        },
-      ],
     };
   },
+  computed: {
+    numFailed(): number {
+      return this.rows.length;
+    },
+    numMuted(): number {
+      return this.rows.filter(row => row.mute).length;
+    },
+    friendlyTimeLastRun(): string {
+      return dayjs().to(dayjs(this.timeLastRun));
```

Comment:
Maybe we should just put in `last updated ${ this.timeLastRun.toLocaleString(); }` and drop the `ago` bit because this isn't a web page.

Reply:
I did notice when I switched tabs and returned to the Diagnostics tab that the time difference was in minutes, so that does trigger an update on the view without updating the underlying time stamp.

Note that when we actually return live data, the time stamp will reflect the actual time it was checked.

And that brings up the issue of how we're going to avoid using `setTimeout` etc. -- we can put watchers on directories and files, but some other checks aren't watchable, like network connectivity.

Reply:
> I did notice when I switched tabs and returned to the Diagnostics tab that the time difference was in minutes, so that does trigger an update on the view without updating the underlying time stamp.

Huh, I couldn't observe that (I thought we essentially always triggered a check on switching to the page, so it's always _a few seconds ago_).

> And that brings up the issue of how we're going to avoid using `setTimeout` etc. -- we can put watchers on directories and files, but some other checks aren't watchable, like network connectivity.

I thought we're just going to let the user hit the recheck button for those. (There probably isn't even a need for watchers?)

Reply:
I don't know how I saw something like '3 minutes ago' yesterday -- I can't repro that. No point doing anything about this while we're still hardwiring data